src/components/data/SavedColumnSettings.js

```diff
@@ -222,9 +222,8 @@ const SavedColumnSettings = ({ workspace, snapshotName, entityType, entityMetada
       }, 'Save this column selection')
     ]),
 
-    hr({ style: { margin: '1rem 0' } }),
-
     _.size(savedColumnSettings) > 0 && h(Fragment, [
+      hr({ style: { margin: '1rem 0' } }),
```

Comment:
This is unrelated, but there's no need to show this divider if there is nothing below it.